content/systems/filecoin_nodes/node_types/_index.md

```diff
@@ -3,9 +3,96 @@ title: Node Types
 weight: 1
 bookCollapseSection: true
 dashboardWeight: 1
-dashboardState: wip
+dashboardState: stable
 dashboardAudit: n/a
 dashboardTests: 0
 ---
 
-# Node Types
\ No newline at end of file
+Nodes in the Filecoin network are primarily identified in terms of the services they provide. The type of node, therefore, depends on which services a node provides. A basic set of services in the Filecoin network include:
+- chain verification
+- storage market client
+- storage market provider
+- retrieval market client
+- retrieval market provider
+- storage mining
+
+Any node participating in the Filecoin network should provide the _chain verification_ service as a minimum. Depending on which extra services a node provides on top of chain verification, it gets the corresponding functionality and Node Type "label".
+
+Nodes can be realized with a repository (directory) in the host in a one-to-one relationship - that is, one repo belongs to a single node. That said, one host can implement multiple Filecoin nodes by having the corresponding repositories.
+
+A Filecoin implementation should support the following subsystems, or types of nodes:
```

Comment:
I
```suggestion
A Filecoin implementation can support the following subsystems, or types of nodes:
```

Reason for can instead of should is that there is no requirement that any implementation provide all these nodes. Some of the other language implementations will definitely not provide all these at launch. "Chain Verifier Node" is really the only MVP needed to be valid filecoin implementation.